app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/ContributorsActivity.java

```diff
@@ -51,8 +54,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.item_repository:
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/DroidKaigi/conference-app-2017"));
-                startActivity(intent);
+                navigator.navigateToWebPage("https://github.com/DroidKaigi/conference-app-2017");
```

Comment:
How about handling option item selected event in `Fragment` like [this](https://github.com/DroidKaigi/conference-app-2017/blob/7f26024d1ee5c1985fbfc223af99c1e53fff7f89/app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java#L98) ❓